mypyc/test-data/irbuild-set.test

```diff
@@ -799,3 +799,228 @@ L4:
     r11 = CPy_NoErrOccurred()
 L5:
     return 1
+
+[case testSetFromRange1]
+def fn() -> set[int]:
```

Comment:
I was able to find some example of `set(range(...))`, but only a few. I think the first two irbuild test cases are sufficient, due to maintainability concerns. These would be reasonable as run tests (preferable as new cases in an existing test case).